run-validation/flow_validation.py

```diff
@@ -583,10 +583,11 @@ def main(flow_file, charge_only):
             fractions = flow_h5['mc_truth/packet_fraction/data']['fraction']
             summed_fractions = fractions.sum(axis=-1)
             fig, ax = plt.subplots(constrained_layout = True)
-            ax.hist(summed_fractions, bins = np.arange(0.895,1.1,0.01), density = True)
+            ax.hist(summed_fractions, bins= np.arange(-0.05, summed_fractions.max(), 0.1))
```

Comment:
@krwood Re: previous comment
Not sure if you saw the update, but here I changed here to have the upper bound be determined by the maximum value, so it should capture the largest value.